bin/setup.sh

```diff
@@ -72,4 +80,7 @@ echo "Updating WordPress"
 docker-compose exec --user www-data phpfpm wp core update
 docker-compose exec --user www-data phpfpm wp core update-db
 
+#insert url on hosts file
+echo "127.0.0.1 $URL" | sudo tee -ai /private/etc/hosts
```

Comment:
This should work well on windows ;)

Reply:
:| pfff nevermind. Just ignore this comment.

Reply:
Thinking about it again. I don't think that this should be "forced" on the hosts. At least I don't think that the setup.sh should be the responsible party for this. I would remove it.

Reply:
What other option do we have? Writing manually?